pom.xml

```diff
@@ -59,21 +66,27 @@
 
     <build>
         <pluginManagement>
-          <plugins>
+            <plugins>
+                <plugin>
+                    <groupId>com.itemis.maven.plugins</groupId>
+                    <artifactId>unleash-maven-plugin</artifactId>
+                    <version>2.7.2</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>com.itemis.maven.plugins</groupId>
+                            <artifactId>unleash-scm-provider-git</artifactId>
+                            <version>2.1.0</version>
+                        </dependency>
+                    </dependencies>
+                </plugin>
+            </plugins>
+        </pluginManagement>
```

Comment:
This block could be removed as the unleash plugin is already available within the parent pom.